config.yaml

```diff
@@ -45,3 +45,8 @@ options:
     description: |
       If this is set, update-status will check if this node is currently validating for this address.
       HINT: if this config parameter is set, the actions is-validating-this-era and is-validating-next-era can be used.
+  parachain-spec-url:
```

Comment:
Nit: "relay chains", e.g. live networks or similar, can also use spec files so perhaps the name isn't 100% on point.

`spec-file-url` instead perhaps?

Reply:
Or two params. `relaychain-spec-url` and `parachain-spec-url`. So that it is possible to use both or just one of them. I guess for live chains it would not matter which one is used.

Reply:
A live chain is more like a relaychain than a parachain though? Since it does not "have" a relaychain "above".

Reply:
It might not be possible to call a live chain either.
Parachain: it has a relaychain
Relaychain: it is supporting other chains (parachains)
Live: it becomes a parachain if it is hooked up on a relaychain. It might also become a relaychain if starting to support other chains? I don't know, have never seen this situation though.

For parachain it becomes clear what to use, but for relay and live I'm not sure...

1. Parachain (two spec files): `chain-spec-url` and `relaychain-spec-url`
2. Relaychain (one spec file): which one to use?
3. Live chain (one spec file): which one to use?

Reply:
> It might not be possible to call a live chain either. Parachain: it has a relaychain Relaychain: it is supporting other chains (parachains) Live: it becomes a parachain if it is hooked up on a relaychain. It might also become a relaychain if starting to support other chains? I don't know, have never seen this situation though.
> 
> For parachain it becomes clear what to use, but for relay and live I'm not sure...
> 
> 1. Parachain (two spec files): `chain-spec-url` and `relaychain-spec-url`
> 2. Relaychain (one spec file): which one to use?
> 3. Live chain (one spec file): which one to use?

So after reading the code it's like this now, right?:

1. Parachain (two spec files): `chain-spec-url` and `relaychain-spec-url`
2. Relaychain (one spec file): `chain-spec-url` or  `relaychain-spec-url`
3. Live chain (one spec file): `chain-spec-url` or  `relaychain-spec-url`

Reply:
Or no I'm wrong. Setting `relaychain-spec-url` for 1 and 2 would add an `--` to the service args. That could be a bit confusing, that for a relaychain one must use `chain-spec-url` and not `relaychain-spec-url`. Am I right?

Reply:
> Would it be enough to call the config local-relaychain-spec-url and make this clear in the description or is it still risk for confusion or do we want to reconsider the design again?

I like this option. Makes it clear that it's only when a local relay should run that it should be used. Could also make this clear in the config, e.g. "do not use when running a relaychain or live network",

Reply:
Sounds good!